inc/opendefs.h

```diff
@@ -158,6 +158,9 @@ enum {
    COMPONENT_TOHLONE                   = 0x22,
    COMPONENT_UECHO                     = 0x23,
    COMPONENT_RRT                       = 0x24,
+   //START OF TELEMATICS CODE

```

Comment:
don't put comments where you start/stop.
If everyone did this, it would be a nightmare. It's important your name appears as a contributor: in each file you changes, just add you name in the description of the file on the top